src/Simulation/EntryPointDriver/DriverSettings.cs

```diff
@@ -63,6 +69,8 @@ public sealed class DriverSettings
             string quantumSimulatorName,
             string toffoliSimulatorName,
             string resourcesEstimatorName,
+            //string sparseSimulatorName,   // TODO(rokuzmin): Requires support in Q# compiler? 
+                                            // Is invoked from the .cs code generated from .qs, and the arg for this param is not provided.
```

Comment:
@samarsha, @bettinaheim, please advise regarding this fragment and please confirm that the changes in `EntryPointDriver` are correct.

Reply:
To be postponed to a subsequent PR.